arches_controlled_lists/src/arches_controlled_lists/components/misc/ControlledListSplash.vue

```diff
@@ -18,15 +18,23 @@ const { $gettext } = useGettext();
         </div>
 
         <div class="controlled-list-splash-description">
-            {{ $gettext("Select a list from the sidebar.") }}
+            <div>
+                {{ $gettext("Use the Controlled List Manager to create new lists of controlled vocabularies, ") }}
+            </div>
+            <div>
+                {{ $gettext("or update an existing list by selecting it from the tree in the sidebar.") }}
+            </div>
```

Comment:
We cannot reliably translate strings broken into separate gettext calls. Can this be done a different way?